apps/condo/domains/marketplace/schema/MarketItemFile.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Generated by `createschema marketplace.MarketItemFile 'marketItem:Relationship:MarketItem:CASCADE; file:File;'`
+ */
+
+const get = require('lodash/get')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
+const { getFileMetaAfterChange } = require('@condo/domains/common/utils/fileAdapter')
+const access = require('@condo/domains/marketplace/access/MarketItemFile')
+const { addOrganizationFieldPlugin } = require('@condo/domains/organization/schema/plugins/addOrganizationFieldPlugin')
+
+
+const MARKET_ITEM_FILE_FOLDER_NAME = 'marketitemfile'
+const Adapter = new FileAdapter(MARKET_ITEM_FILE_FOLDER_NAME)
+const fileMetaAfterChange = getFileMetaAfterChange(Adapter)
+const ALLOWED_MIME_TYPES = [
+    'image/jpeg',
+    'image/png',
+    'image/heic',
+]
+const ERRORS = {
+    NON_IMAGE_FILE: {
+        code: BAD_USER_INPUT,
+        type: 'NON_IMAGE_FILE',
+    },
```

Comment:
Try something like this to keep all error fields together.
```suggestion
    FORBIDDEN_FILE_TYPE: (mimeType) => ({
        code: BAD_USER_INPUT,
        type: 'NON_IMAGE_FILE',
        message: `Expected file to be one of the following mimetypes: ${ALLOWED_MIME_TYPES.map(type => `"${type}"`).join(', ')}. But got: ${mimetype}`,
    }),
```